qiskit_experiments/library/randomized_benchmarking/clifford_utils.py

```diff
@@ -229,16 +294,322 @@ def _unpack_num(num, sig):
             num //= k
         return res
 
-    @staticmethod
-    def _unpack_num_multi_sigs(num, sigs):
-        """Returns the result of `_unpack_num` on one of the
-        signatures in `sigs`
-        """
-        for i, sig in enumerate(sigs):
-            sig_size = 1
-            for k in sig:
-                sig_size *= k
-            if num < sig_size:
-                return [i] + CliffordUtils._unpack_num(num, sig)
-            num -= sig_size
-        return None
+
+# Constant mapping from 1Q single Clifford gate to 1Q Clifford numerical identifier.
+# This table must be generated using `data.generate_clifford_data.gen_cliff_single_1q_gate_map`.
+_CLIFF_SINGLE_GATE_MAP_1Q = {
+    ("id", (0,)): 0,
+    ("h", (0,)): 1,
+    ("sxdg", (0,)): 2,
+    ("s", (0,)): 4,
+    ("x", (0,)): 6,
+    ("sx", (0,)): 8,
+    ("y", (0,)): 12,
+    ("z", (0,)): 18,
+    ("sdg", (0,)): 22,
+}
+# Constant mapping from 2Q single Clifford gate to 2Q Clifford numerical identifier.
+# This table must be generated using `data.generate_clifford_data.gen_cliff_single_2q_gate_map`.
```

Comment:
I think it would be helpful to add that the thing defining the ordering is really the layers, e.g.
```
# Constant mapping from 2Q single Clifford gate to 2Q Clifford numerical identifier.
# This table must be generated using `data.generate_clifford_data.gen_cliff_single_2q_gate_map`, or, 
# equivalently, correspond to the ordering defined by _layer_indices_from_num and _CLIFFORD_LAYER.
```